Add notations page (#377)
aramis-lab/leaspy: docs/notations.md

# Notations

This page contains the definitions and notations of the variables used in Leaspy.

## Notations, concepts, and names in DAG

The following table displays the relationships between a variable mathematical notation, the associated concept, and the string name used in the DAG if this variable is handled this way.

| Notation   | Concept                                                | Name in the DAG |
|------------|--------------------------------------------------------|-----------------|

> | $ \tau_i $ | [estimated reference time](#estimated-reference-time)  | ???             |

Reviewer: ```suggestion
| $ \tau_i $ | [estimated reference time](#estimated-reference-time)  | `tau`            |
```

> |  $ e^{\xi_i} $ | [individual speed factor](#individual-speed-factor) | ???             |

Reviewer: We don't have a specific parameter for this. It can be calculated with exponential xi or we can define a new parameter

Author: I went quite fast on this, maybe the individual speed factor is $ \xi_i $ and not its exponential ? Maybe @JulietteOrtholand knows better ?

Reviewer 2: I made a suggestion and discuss it below, I close this thread.

> |  $ e^{\xi_i} $ | [individual speed factor](#individual-speed-factor) | ???             |

Reviewer 2: ```suggestion
|  $ \xi_i $ | [individual log speed factor](#individual-speed-factor) | `xi`             |
```

$ \alpha_i = e^{\xi_i} $ does not exist in the DAG I think, maybe it could be added (related to Esther questions) when moving time reparametrisation in the classes issues.

> |  $ \xi_i $ | [individual log speed factor](#individual-speed-factor) | `xi`             |

Author: ```suggestion
|  $ \xi_i $ | [individual log speed factor](#individual-log-speed-factor) | `xi`             |
```

| $ \gamma_i(t) $ | [individual trajectory](#individual-trajectory) | ???             |
| $ \psi_i(t) $ | [latent disease age](#latent-disease-age) | ???             |
| $ \mathbf{A} $ | [mixing matrix](#mixing-matrix) | `mixing_matrix` |
| $ t_0 $ | [population reference time](#population-reference-time) | ???             |
| $ \mathbf{s}_i$ | [sources](#sources) | `sources` |
| $ w_{i,k} $ | [space shift](#space-shift) | `space_shifts`  |

> | $ u_{i,l} $ | [survival shift](#survival-shift) | ???             |

Reviewer: @JulietteOrtholand is it `nu`?

> | $ \gamma_i(t) $ | [individual trajectory](#individual-trajectory) | ???             |
> | $ \psi_i(t) $ | [latent disease age](#latent-disease-age) | ???             |
> | $ \mathbf{A} $ | [mixing matrix](#mixing-matrix) | `mixing_matrix` |
> | $ t_0 $ | [population reference time](#population-reference-time) | ???             |
> | $ \mathbf{s}_i$ | [sources](#sources) | `sources` |
> | $ w_{i,k} $ | [space shift](#space-shift) | `space_shifts`  |
> | $ u_{i,l} $ | [survival shift](#survival-shift) | ???             |

Reviewer 2: ```suggestion
| $ \gamma_i(t) $ | [individual trajectory](#individual-trajectory) | `model`             |
| $ \psi_i(t) $ | [latent disease age](#latent-disease-age) | `rt`             |
| $ \mathbf{A} $ | [mixing matrix](#mixing-matrix) | `mixing_matrix` |
| $ t_0 $ | [population reference time](#population-reference-time) | `tau_mean`             |
| $ \mathbf{s}_i$ | [sources](#sources) | `sources` |
| $ w_{i,k} $ | [space shift](#space-shift) | `space_shifts`  |
| $ u_{i,l} $ | [survival shift](#survival-shift) | `survival_shifts`             |
```

## Concept definitions

This section explains each mathematical concept and provides mathematical definitions when needed.

### Estimated reference time

> The *estimated reference time* for a given individual $ i $ is denoted as $ \tau_i $ and is defined as **TODO**.

Reviewer 2: ```suggestion
The *estimated reference time* for a given individual $ i $ is denoted as $ \tau_i $. It follows $\tau_i \sim \mathcal{N}(t_0, \sigma^2_{\tau})$.
```

> ### Individual speed factor
> 
> The *individual speed factor* for a given individual $ i $ is denoted as $ e^{\xi_i} $ and is defined as **TODO**.

Reviewer 2: ```suggestion
### Individual log speed factor

The *individual log speed factor* for a given individual $ i $ is denoted as $ \xi_i $ . It follows $\xi_i \sim \mathcal{N}(0, \sigma^2_{\xi})$.
```

### Individual trajectory

> The *individual trajectory* for a given individual $ i $ is denoted as $ \gamma_i(t) $ and represents **TODO**.

Reviewer 2: ```suggestion
The *individual trajectory* for a given individual $ i $ is denoted as $ \gamma_i(t) $ and represents the disease progression of the patient $i$. It can be indexed by $k$ when $K$ outcomes are estimated.  
```

### Latent disease age

> The *latent disease age*, for a given individual $ i $, is denoted as $ \psi_i(t) $.
> 
> It represents **TODO** and is defined as:
> 
> $$
> \psi_i(t) = e^{\xi_i}(t - \tau_i) + t_0
> $$
> 
> where :
> 
> - $ e^{\xi_i} $ is the [individual speed factor](#individual-speed-factor) of individual $ i $.
> - $ \tau_i $ is the [estimated reference time](#estimated-reference-time) of individual $ i $.
> - $ t_0 $ is the [population reference time](#population-reference-time).

Reviewer 2: ```suggestion
The *latent disease age*, for a given individual $ i $, is denoted as $ \psi_i(t) $.

It represents a transformation from chronological time $t$ to latent disease age that is related to its stage in the disease and is defined as:

$$
\psi_i(t) = e^{\xi_i}(t - \tau_i) + t_0
$$

where :

- $ e^{\xi_i} $ is the [individual speed factor](#individual-speed-factor) of individual $ i $.
- $ \tau_i $ is the [estimated reference time](#estimated-reference-time) of individual $ i $.
- $ t_0 $ is the [population reference time](#population-reference-time).
```
